tests/Aspire.Hosting.Tests/Aspire.Hosting.Tests.csproj

```diff
@@ -8,11 +8,18 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Aspire.Hosting\Aspire.Hosting.csproj" />
-    <ProjectReference Include="..\..\src\Aspire.Hosting.Azure\Aspire.Hosting.Azure.csproj" />
-    <ProjectReference Include="..\testproject\TestProject.AppHost\TestProject.AppHost.csproj" />
-    
+    <ProjectReference Include="..\..\src\Aspire.Hosting\Aspire.Hosting.csproj" AspireHostLibrary="true" />
+    <ProjectReference Include="..\..\src\Aspire.Hosting.Azure\Aspire.Hosting.Azure.csproj" AspireHostLibrary="true" />
+    <ProjectReference Include="..\testproject\TestProject.AppHost\TestProject.AppHost.csproj" AspireHostLibrary="true" />
+
+    <ProjectReference Include="..\..\src\Components\Aspire.Microsoft.Data.SqlClient\Aspire.Microsoft.Data.SqlClient.csproj" AspireHostLibrary="true" />
+    <ProjectReference Include="..\..\src\Components\Aspire.MongoDB.Driver\Aspire.MongoDB.Driver.csproj" AspireHostLibrary="true" />
+    <ProjectReference Include="..\..\src\Components\Aspire.MySqlConnector\Aspire.MySqlConnector.csproj" AspireHostLibrary="true" />
+    <ProjectReference Include="..\..\src\Components\Aspire.Npgsql\Aspire.Npgsql.csproj" AspireHostLibrary="true" />
+    <ProjectReference Include="..\..\src\Components\Aspire.Oracle.EntityFrameworkCore\Aspire.Oracle.EntityFrameworkCore.csproj" AspireHostLibrary="true" />
+
     <PackageReference Include="Microsoft.Extensions.Http.Resilience" />
```

Comment:
Should this have `Version="$(MicrosoftExtensionsHttpResiliencePackageVersion)"`. And similarly a fixed version for `Polly` too?

Reply:
We use CPM (central package management) in Directory.Packages.props in the root of the repo.

Reply:
ah, thank you!